Week-3/responsiveness_and_bootstrap/Bootstrap for a vacation website/script.js

```diff
@@ -0,0 +1,12 @@
+function showAlert(alertId) {
+   const bootstrapAlert = document.querySelector("#" + alertId);
+   const collapse = new bootstrap.Collapse(bootstrapAlert); 
+   collapse.show();
+}
+
+const modalYesBtn = document.querySelector("#contact-modal .btn-primary");
+modalYesBtn.addEventListener("click", function () {    
+   const sendBtn = document.querySelector("#send-message-btn");
```

Comment:
you can use document.getElementById here which is very very faster then the query selector.